spec/dashboard_spec.js

```diff
@@ -44,12 +44,15 @@ describe('dashboard', () => {
         monitoring.nextSearches();
         monitoring.nextReorder();
         monitoring.saveSettings();
-        browser.wait(() =>
-            dashboard.getWidget(0)
-                .element(by.id('title'))
-                .isDisplayed(),
-            5000
-        );
+
+        // TODO
+        // this second saving is needed for passing tests on CI,
+        // I could reproduce this scenario locally once, so probably
+        // there is can be issue with first saving...
+        // Same thing was in line 74, so probably someone got this before too
+        dashboard.showMonitoringSettings(0);
+        monitoring.saveSettings();
```

Comment:
just wondering if this is still needed with nav?

Reply:
I think so, when I got this scenario locally once I didn't refresh the page...

I'll check once again on Monday.

Reply:
I've just checked and this's still needed.